src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java

```diff
@@ -110,9 +110,7 @@ public void launch(SlaveComputer computer, TaskListener listener) {
         try {
             KubernetesClient client = slave.getKubernetesCloud().connect();
             Pod pod = template.build(slave);
-            if (template.isShowRawYaml()) {
```

Comment:
Related, or just another bug?

Reply:
Not a bug, but with the impl change, having the pod at hand is mandatory, so this is combined with https://github.com/jenkinsci/kubernetes-plugin/pull/717/files#diff-7de256a4d98e2297b5323f04b76bd471R410